_includes/manuals/nightly/2.1_quickstart_installation.md

```diff
@@ -58,6 +60,17 @@ sudo yum -y install https://yum.puppet.com/puppet6-release-el-7.noarch.rpm
 {% endhighlight %}
 </div>
 
+<div class="quickstart_os quickstart_os_rhel8 quickstart_os_el8">
```

Comment:
Should this step also point to the now required "configmanagement-ansible" repo? Or is that optional due to the fact only a plugin needs ansible?

Reply:
The latter I should think.

Reply:
wfm.